docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,331 @@
+# Service Catalog Demonstration Walkthrough
+
+This document outlines the basic features of the service catalog by walking
+through a basic use case.
+
+## Step 1 - Installing the Service Catalog System
+
+The service catalog is conveniently packaged as a [helm](http://helm.sh/) 
+chart for installation.
+
+The chart is located in the [charts/catalog](../charts/catalog) directory in this repository, and 
+supports a wide variety of customizations which are laid out in the README.md in that
+directory. To install the service-catalog with sensible defaults, execute this command:
+
+```console
+helm install charts/catalog --name catalog --namespace catalog
```

Comment:
I talked to @krancour about this and he thought we should mention that you need to do

```
helm install charts/catalog --name catalog --namespace catalog --set "apiserver.service.type=NodePort"
```
if you are running in minikube.

(Currently the default is actually NodePort but from our discussion it seems that we want to change that to LoadBalancer which matches the readme)